Automattic/woocommerce-payments
# Branding rollout: WooCommerce Payments -> WooPayments

Fixes #6549 

#### Changes proposed in this Pull Request

This PR modifies user-facing places with WooCommerce Payments occurrences in order to change it for WooPayments. This is a **plugin** part of the Branding Rollout Project paJDYF-7Sw-p2.

**Disclaimer**: WordPress.org logo for WCPay (see below) will be changed in #6669. 
<img src="https://github.com/Automattic/woocommerce-payments/assets/79862886/1db520c7-df08-4022-a00d-539a7efdc49a" width="600"/>

- Changed brand name in the changelog
- Also, it introduces small refactoring of translatable strings as suggested by @vladolaru https://github.com/Automattic/woocommerce-payments/pull/6581#issuecomment-1621479947

#### Testing instructions

* Checkout to `add/6549-branding-rollout`
* Navigate to `WooCommerce -> Payments -> All payment methods`. Check that all _WooCommerce Payments_ mentions are changed to _WooPayments_.
* Navigate to `WooCommerce -> Payments -> WooPayments`. Check that all occurrences of _WooCommerce Payments_ are modified to _WooPayments_ including logo. Click `Advanced settings` in the bottom of the page and check it too.
  ![image](https://github.com/Automattic/woocommerce-payments/assets/79862886/d1eb02d7-c4f0-4a17-b737-7a2f3be8d1af)
* Check that text in notices is also modified
  ![image](https://github.com/Automattic/woocommerce-payments/assets/79862886/e6d19cbc-8b8f-4b20-982e-a88d95cd9315)
* Review the code and see if all the changes make sense and everything looks safe to modify (does not have dependencies). See p1688400047819289-slack-CGGCLBN58 for more context.

-------------------

- [x] Run `npm run changelog` to add a changelog file, and choose `patch` to leave it empty if the change is insignificant. You can add multiple changelog files in one PR by running this command a few times. 

**Post merge**

- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-6.2.0#branding-rollout-woocommerce-payments---woopayments
